### PR TITLE
PHPLIB-1116: Add tests for runCommand spec

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/PHPLIB-1116

Opening this PR preemptively as I noticed some test runner changes were necessary to POC https://github.com/mongodb/specifications/pull/1389. This PR will be updated once the spec is merged and we have a final sync commit.

### CODE REVIEW

**Comment by jmikola** [comment]
This PR should wait on https://github.com/mongodb/specifications/pull/1410.

File: tests/UnifiedSpecTests/Operation.php [hunk 1]
```diff
@@ -683,10 +683,16 @@ function (CollectionInfo $info) {
...
-                )->toArray()[0];
+                // Note: commandName is not used by PHP
+                $options = array_diff_key($args, ['command' => 1, 'commandName' => 1]);
+
+                /* runCommand spec tests may execute commands that return a
+                 * cursor (e.g. aggregate). PHPC creates a cursor automatically
+                 * so cannot return the original command result. Existing spec
+                 * tests do not evaluate the result, so we can return null here.
+                 * If that changes down the line, we can use command monitoring
+                 * to capture and return the command result. */
```
Comment (alcaeus) [inline tests/UnifiedSpecTests/Operation.php:694]: 👍

**Review by alcaeus — APPROVED** [review]

**Comment by jmikola** [comment]
Updated with follow-up commit for [DRIVERS-2577](https://jira.mongodb.org/browse/DRIVERS-2577): https://github.com/mongodb/specifications/commit/146964a452f2d981f9b6b6d6063703038ed44d13

Will merge after CI completes.

**Comment by jmikola** [comment]
Found a separate issue with the spec tests, so this will need to wait on https://github.com/mongodb/specifications/pull/1416 to be resolved. Pushing a POC commit in the meantime.

**Pull request merged**